### DISCUSSION TITLE
Am I correct in surmising that there's no simple way to use Docker to run Medusa inside of a container?

### DISCUSSION THREAD

**Discussion by Raytle** [medusajs/medusa#6045] [Q&A]
I am not complaining, rather I am explaining when I indicate that it seems like it's trivial to use Docker to run [Vendure](https://docs.vendure.io/guides/deployment/using-docker/) inside of container.

I read the following...

1. https://github.com/medusajs/medusa/discussions/4984
2. https://github.com/medusajs/docker-medusa/?tab=readme-ov-file#please-note
3. https://github.com/medusajs/docker-medusa/?tab=readme-ov-file#requirements

I paid close attention to the following (emphasis added)...

This repo is managed by the Medusa Community. **Medusa does _not_ provide official support for Docker**, but we will accept fixes and documentation. Use at your own risk.

**This project is inteded [sic] for _development_ only at this time.**

**Comment by olivermrbl** [comment]
Thanks for showing interest @Raytle. 

There is likely a simple way to use Docker to run Medusa, as all you need is a Node server, Postgres, and Redis for a full setup. All services should be straightforward to configure in a compose file. 

The comment in the `docker-medusa` repository should be understood as; "we don't officially support a Docker setup with Medusa to avoid the possible maintenance burden that could come from it". You could argue that if the setup is simple, why not take on the burden of supporting it officially? We tried this previously but came to realize that many developers out there are much more knowledgeable about and competent in Docker setups than we are, so we decided to let it be up to the community to help each other out. 

As you'll find in our Discord, there are plenty of users using Docker with Medusa alreadt. I highly recommend you ask for help there or explore old chat messages to find what you are looking for.

I am unsure if this answers your question, but I hope it clarifies some things about Docker and Medusa.

**Comment by Raytle** [comment]
> We tried this previously but came to realize that many developers out there are much more knowledgeable about and competent in Docker setups than we are, so we decided to let it be up to the community to help each other out.

My duuuuuude. That straw man argument is rubbish. Er, uh, I mean, please don't proffer inane arguments to me. Specifically, don't conflate documentation with support. Sure, they often go hand in hand, but they don't have to. In other words, it is entirely reasonable to document things you don't  support.

You guys don't need to "officially support using Docker" in order to a create a video (screencast) showing how to use Docker to install Medusa in a container.

Here. Let me help you write the beginning of a script you might use....

"Hi everyone. Although we do not offer any support whatsoever for using Docker to install Medusa in a container; nonetheless, in this video we show how to use Docker to install Medusa in a container. In other words, yes, we are willing to show you; but, no, we aren't willing to help you use Docker to install Medusa in a container."

I am confident that I have destroyed your demonstrably feeble argument. Nonetheless, feel free to rebut mine, for I am accustomed to arguing with ignorant engineers who wantonly violate the laws of formal logic.

If you guys are too busy to create such a video (screencast), perhaps you might offer one of your favorite users, say, 100 US dollars to create a 1/2 hour (30 minute) video (screencast) for you.

Also, if you think most potential users will "just hop on Discord" to ask questions then I think you are deluding yourselves. Personally, I loathe Discord. I "just hoped on" another company's Discord a couple of times about five years ago to ask some questions about a software application. It was a big waste of time for me.

Believe it or not, when I see a link to Discord on a company's homepage, it normally raises a major red flag in my mind because I think, "This company is run by a bunch of ignorant techies who adore prattling on about their beloved technical product, but couldn't be bothered to properly document/explain it in videos. I hate dealing with these sorts of companies."
